src/messageComposer/middleware/textComposer/commands.ts

```diff
@@ -143,29 +148,49 @@ export const createCommandsMiddleware = (
           return next(newState);
         }
 
+        const inputText = triggerWithToken?.toLowerCase().slice(1);
+        const matchedCommand = commands.find((command) => {
+          if (!command.name || !inputText) return false;
+          return isTextMatched(inputText, command.name.toLowerCase());
+        });
+
+        if (matchedCommand) {
```

Comment:
Is this condition necessary? The command can be passed to the complete below it.

Reply:
Resp. we could remove the complete call and just use next